src/vue/forms/InvestForm.vue

```diff
@@ -220,6 +220,7 @@ import { mapGetters, mapActions } from 'vuex'
 import { vuexTypes } from '@/vuex'
 import { vueRoutes } from '@/vue-router/routes'
 import { MathUtil } from '@/js/utils'
+import { keyValues } from '@/key-values'
```

Comment:
can't we use this.$kv?

Reply:
and everywhere you import `keyValues` to component

Reply:
if you import like this, your IDE autocompletes the `keyValues` properties. Unfortunately, `this.$kv` does not autocomplete. I would be glad to migrate to `this.$kv` but autocomplete is much sweeter